src/bdc/steps/regionalatlas.py

```diff
@@ -6,48 +6,49 @@
 import geopandas as gpd
 import osmnx
 import pandas as pd
+from geopandas.tools import sjoin
 from pandas import DataFrame
-from shapely.geometry import Point
+from shapely.geometry import Point, shape
 from tqdm import tqdm
 
 from bdc.steps.step import Step
 from logger import get_logger
-from shapely.geometry import shape
-from geopandas.tools import sjoin
 
 log = get_logger()
 
 
 class RegionalAtlas(Step):
     name: str = "Regional_Atlas"
-    reagionalatlas_feature_keys = {
-        "pop_density":"ai0201",
-        "pop_development":"ai0202",
-        "age_0":"ai0203",
-        "age_1":"ai0204",
-        "age_2":"ai0205",
-        "age_3":"ai0206",
-        "age_4":"ai0207",
-         "pop_avg_age (Bevölkerung)":"ai0218",
-        "per_service_sector":"ai0706",        
-        "per_trade":"ai0707",
-        "employment_rate (Beschäftigt)":"ai0710",
-        "unemployment_rate (Arbeitslos)":"ai0801",
-        "per_long_term_unemployment":"ai0808",
-        "investments_p_employee":"ai1001",
-        "gross_salary_p_employee":"ai1002",
-        "disp_income_p_inhabitant":"ai1601",        
-        "tot_income_p_taxpayer":"ai1602",
-        "gdp_p_employee":"ai1701",
-        "gdp_development":"ai1702",                       
-        "gdp_p_inhabitant":"ai1703",
-        "gdp_p_workhours":"ai1704",   
-        "pop_avg_age (Gesamtbevölkerung)":"ai_z01",
-        "unemployment_rate (Erwerbslos)":"ai_z08"
+    reagionalatlas_feature_keys: dict = {
+        "pop_density": "ai0201",
+        "pop_development": "ai0202",
+        "age_0": "ai0203",
+        "age_1": "ai0204",
+        "age_2": "ai0205",
+        "age_3": "ai0206",
+        "age_4": "ai0207",
+        "pop_avg_age (Bevölkerung)": "ai0218",
+        "per_service_sector": "ai0706",
+        "per_trade": "ai0707",
+        "employment_rate (Beschäftigt)": "ai0710",
+        "unemployment_rate (Arbeitslos)": "ai0801",
+        "per_long_term_unemployment": "ai0808",
+        "investments_p_employee": "ai1001",
+        "gross_salary_p_employee": "ai1002",
+        "disp_income_p_inhabitant": "ai1601",
+        "tot_income_p_taxpayer": "ai1602",
+        "gdp_p_employee": "ai1701",
+        "gdp_development": "ai1702",
+        "gdp_p_inhabitant": "ai1703",
+        "gdp_p_workhours": "ai1704",
+        "pop_avg_age (Gesamtbevölkerung)": "ai_z01",
+        "unemployment_rate (Erwerbslos)": "ai_z08",
     }
 
-    df_fields: list[str] = reagionalatlas_feature_keys.values()
-    #germany_gdf = osmnx.geocode_to_gdf("Germany")
+    added_cols: list[str] = reagionalatlas_feature_keys.keys()
+    regions_gdfs = gpd.GeoDataFrame()
+    empty_result: dict = dict.fromkeys(added_cols)
+    # germany_gdf = osmnx.geocode_to_gdf("Germany")
 
```

Comment:
Please add the required_cols
```suggestion
required_cols = ["google_places_formatted_address"]
```